vocabsieve/forvo.py

```diff
@@ -43,35 +47,32 @@ def get_pronunciations(self) -> Forvo:
         else:
             raise Exception("failed to fetch forvo page")
         html = BeautifulSoup(page, "lxml")
-        available_langs_els = html.find_all(
-            id=re.compile(r"language-container-\w{2,4}"))
-        available_langs = [
-            re.findall(
-                r"language-container-(\w{2,4})",
-                el.attrs["id"])[0] for el in available_langs_els]
-        if self.language not in available_langs:
+        available_pronunciations_lists_el = html.find_all(
+            id=re.compile(r"^pronunciations-list-[a-z]+(?:_[a-z]+)?+"))
+        available_lang_locales = [
+            self.get_locale(*re.findall(
+                r"^pronunciations-list-([a-z]+)(?:_([a-z]+))?",
+                el.attrs["id"])[0]) for el in available_pronunciations_lists_el]
```

Comment:
Converts a list of tuples it finds from the regex (`[("pt", "br"), ("sp", "")]`) into a list of strings e.g. `["pt_br", "sp"]`